waggle-dance-core/pom.xml

```diff
@@ -350,6 +350,21 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
This section is for `test` dependencies. Is this only needed for tests? If so `<scope>test</scope>` should be added. If not it should be moved to the `hive` section between lines 118 and 163.